tests/s2_inference/test_model_registry.py

```diff
@@ -0,0 +1,66 @@
+import unittest
+
+from marqo.s2_inference.model_registry import _get_open_clip_properties, load_model_properties
+from tests.marqo_test import MarqoTestCase
+
+
+class TestModelRegistry(unittest.TestCase):
+    def setUp(self):
+        self.open_clip_properties = _get_open_clip_properties()
+
+    def test_open_clip_model_configurations_match(self):
+        with self.subTest("Check if Marqo/ViT-B-32.openai & open_clip/ViT-B-32/openai configurations match"):
```

Comment:
This is hard to read and maintain. I suggest having a list of pairs to test first, then one loop to test them all. e.g.

models_pairs = [('Marqo/ViT-B-32.openai', 'open_clip/ViT-B-32/openai'), ...]

for pair in model_pairs:
    with self.subTest(...

Furthermore, please add a description to this test otherwise in a few months' time we will have no idea why we have such a test.

Reply:
good idea, incorporated